File: benchmarks/cross/progs/particles.lua
local total = 0.0
for frame = 1, 400 do
  local parts = {}
  for i = 0, 1999 do parts[i+1] = {x=i*1.0, y=i*0.5, life=60} end
  for j = 1, 2000 do total = total + parts[j].x end
end
print(math.floor(total))

File: benchmarks/cross/progs/particles.js
let total=0.0;
for(let f=0;f<400;f++){const parts=[];for(let i=0;i<2000;i++)parts.push({x:i*1.0,y:i*0.5,life:60});for(let j=0;j<2000;j++)total+=parts[j].x;}
console.log(Math.floor(total));
